lib/dpul_collections_web/live/item_live.ex

```diff
@@ -109,24 +109,35 @@ defmodule DpulCollectionsWeb.ItemLive do
         height="800"
       />
 
-      <button class="btn-primary">
-        <a href="#" target="_blank">
-          {gettext("View")}
-        </a>
-      </button>
-
-      <button class="btn-primary">
-        <a
-          href={"#{Application.fetch_env!(:dpul_collections, :web_connections)[:figgy_url]}/catalog/#{@item.id}/pdf"}
-          target="_blank"
-        >
-          {gettext("Download PDF")}
-        </a>
-      </button>
+      <.action_link class="left-arrow-box" href="#" target="_blank">
+        <.icon name="hero-eye" /> {gettext("View")}
+      </.action_link>
+
+      <.action_link
+        href={"#{Application.fetch_env!(:dpul_collections, :web_connections)[:figgy_url]}/catalog/#{@item.id}/pdf"}
+        target="_blank"
+      >
+        <.icon name="hero-arrow-down-on-square" class="h-5" /><span>{gettext("Download")}</span>
+      </.action_link>
     </div>
     """
   end
 
+  slot :inner_block
+  attr :class, :string, default: nil
+  attr :href, :string, default: nil
+  attr :rest, :global, doc: "the arbitrary HTML attributes to add link"
+
+  def action_link(assigns) do
```

Comment:
I'm sort of confused by "action_link" -- it makes me think it's for the action bar. What was the semantic intention here?

Reply:
I dunno. `item_button`? ......`button`?